### PR TITLE
html autobiography info added

### PR DESCRIPTION
**Pull request by abrilmunozc**

### CODE REVIEW

File: 02/index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (eamzea) [inline 02/index.html:6]: This is the title of the file?

Comment (eamzea) [inline 02/index.html:6]: Where is the meta tag we already saw?

File: 02/index.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+</head>
+<body>
+    <p><h1><strong>My autobiography</strong></h1></p>
```
Comment (eamzea) [inline 02/index.html:9]: Why a strong tag inside an h1 tag inside a p tag?

File: 02/index.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+<p>
+    <h2><strong>Name:</strong> <em>Abril Muñoz</em></h2>
+    <strong>Age:</strong> 29 years old
+    <strong>Date of birth:</strong> April <mark>8</mark> 1994
+    <strong>Nationality:</strong> Mexican
+    <strong>Description:</strong> 'I'm Abril, I'm from Mexico, I'm 29 years old, I'm psychologist and I work in a big company called <i>"Adecco"</i>, my currrent role is <i>Tech Recruiter.</i> 
+    I live with my friend, her name is <ins>Viri,</ins> and we live close to the downtown to the City.
+</p>
```
Comment (eamzea) [inline 02/index.html:17]: All this text inside a single paragraph?

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

File: 02/index.html (hunk 1 above)
Comment (abrilmunozc) [reply 02/index.html]: I don't understand the question

File: 02/index.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My autobiography</title>
+</head>
+<body>
+    <p><h1>My autobiography</h1></p>
+    <p><h2>Name: <em>Abril Muñoz</em></h2></p>
```
Comment (eamzea) [inline 02/index.html:10]: Why a heading should be inside a paragraph?

File: 02/index.html [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    <p><strong>Age:</strong> 29 years old</p>
+    <p><strong>Date of birth:</strong> April <mark>8</mark> 1994</p>
+    <p> <strong>Nationality:</strong> Mexican</p>
+    <p><strong>Description:</strong> 'I'm Abril, I'm from Mexico, I'm 29 years old, I'm psychologist and I work in a big company called <i>"Adecco"</i>, my currrent role is <i>Tech Recruiter.</i> 
+        I live with my friend, her name is <ins>Viri,</ins> and we live close to the downtown to the City.</p>
+
```
Comment (eamzea) [inline 02/index.html:16]: I guess you can improve the readability of this short code.

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

**Review by soyedgarzea — APPROVED** [review]

**Pull request merged**